
<launch>
        <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
		<param name="destination_frame" value="base_link"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/scan"/>
                <param name="laserscan_topics" value ="/scan1 /scan2" /> <!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
        <!--Laser scan parameters according to LiDAR specification-->
        <param name="last_laser_topic" value="/scan2"/>
		<param name="angle_max" value="3.14"/>
		<param name="angle_min" value="-3.14"/>
		<param name="range_max" value="30"/><!--When set at 30, max output becomes 31-->
		<param name="range_min" value="0.1"/>
	</node>
</launch>
